test/SingleSelleckt.tests.js

```diff
@@ -1,220 +1,201 @@
-                            itemsClass: 'items',
-                            itemClass: 'item',
-                            selectedClassName: 'isSelected',
-                            highlightClassName: 'isHighlighted'
-                        });
-                    } catch(e){
-                        err = e;
-                    }
+        describe('valid instantation', function(){
 
-                    expect(err).toBeDefined();
-                    expect(err.message).toEqual('Please provide a valid mustache template.');
+            beforeEach(function(){
+                selleckt = new SingleSelleckt({
+                    mainTemplate: template,
+                    itemTemplate: itemTemplate,
+                    $selectEl : $el,
+                    className: 'selleckt',
+                    selectedClass: 'selected',
+                    selectedTextClass: 'selectedText',
+                    itemsClass: 'items',
+                    itemClass: 'item',
+                    selectedClassName: 'isSelected',
+                    highlightClass: 'isHighlighted'
                 });
+            });
 
+            it('stores the selectedClass as this.selectedClass', function(){
+                expect(selleckt.selectedClass).toEqual('selected');
             });
 
-            describe('valid instantation', function(){
+            it('stores the selectedTextClass as this.selectedClass', function(){
+                expect(selleckt.selectedTextClass).toEqual('selectedText');
+            });
 
-                beforeEach(function(){
-                    selleckt = Selleckt.create({
-                        mainTemplate: template,
-                        $selectEl : $el,
-                        className: 'selleckt',
-                        selectedClass: 'selected',
-                        selectedTextClass: 'selectedText',
-                        itemsClass: 'items',
-                        itemClass: 'item',
-                        selectedClassName: 'isSelected',
-                        highlightClass: 'isHighlighted'
-                    });
-                });
+            it('stores the itemsClass as this.itemsClass', function(){
+                expect(selleckt.itemsClass).toEqual('items');
+            });
 
-                it('stores the selectedClass as this.selectedClass', function(){
-                    expect(selleckt.selectedClass).toEqual('selected');
-                });
+            it('stores the itemClass as this.itemClass', function(){
+                expect(selleckt.itemClass).toEqual('item');
+            });
 
-                it('stores the selectedTextClass as this.selectedClass', function(){
-                    expect(selleckt.selectedTextClass).toEqual('selectedText');
-                });
+            it('stores options.mainTemplate as this.template', function(){
+                expect(Mustache.render(selleckt.mainTemplate, {})).toEqual(Mustache.render(template, {}));
```

Comment:
As per my [previous comment](https://github.com/BrandwatchLtd/selleckt/pull/65/files#r24995306) this can be simplified to
```js
expect(selleckt.mainTemplate).toEqual(template);
```
(same for `itemTemplate`)